packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -94,28 +92,13 @@ const AuthHeader = dynamic(
   () => import(/* webpackChunkName: "authHeader" */ './AuthHeader'),
 );
 
-const EmailVerificationSent = dynamic(
-  () =>
-    import(
-      /* webpackChunkName: "emailVerificationSent" */ './EmailVerificationSent'
-    ),
-);
```

Comment:
We just don't use it?

Reply:
Exactly, old code never rendering. Checked here also:

- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729251570944869
- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729256868868289